web/app/js/components/MetricsTable.jsx

```diff
@@ -130,7 +130,7 @@ const columnDefinitions = (resource, namespaces, onFilterClick, showNamespaceCol
       className: "numeric",
       width: smMetricColWidth,
       sorter: (a, b) => numericSort(a.requestRate, b.requestRate),
-      render: d => withTooltip(d, "REQUEST_RATE")
+      render: d => withTooltip(d, "REQUEST_RATE", true)
```

Comment:
do we need the tooltip here? appears to just show the same number, or, `0` if the underlying value is `---`.
![screen shot 2018-09-17 at 11 28 41 am](https://user-images.githubusercontent.com/236915/45642377-da823200-ba6c-11e8-9ba2-6ceb1f849af4.png)

Reply:
the tooltip is intended to show the untruncated value of the number, but it doesn't appear to be super useful seeing as we're also just calculating this from total requests divided by time window, and we don't need this to be super accurate anyways. I can remove the tooltipping